src/skodun/services.py

```
@@ -835,6 +881,10 @@ def cancellation_reason():
             raise
         except BaseException:
             pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)